### DISCUSSION TITLE
Pie Chart Issue - Issue with Label Position [onBorder]

### DISCUSSION THREAD

**Discussion by iamabhi9v** [Abhinandan-Kushwaha/react-native-gifted-charts#528] [General]
Write there's an issue in the pie chart **onBorder** label position.
![Simulator Screen Shot - iPhone 14 - 2024-02-01 at 17 59 52](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/66355946/738c031d-c123-40b3-bcb6-be3da8dbae46)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @iamabhi9v 👋
Thanks for reporting this issue. It has been fixed in the latest version of the library (`1.4.5`)

We have added 2 new props named `paddingHorizontal` and `paddingVertical`. When the label position is **onBorder**, paddingHorizontal and paddingVertical will be automatically assigned values equal to _textBackgroundRadius_. You can override these values by manually passing the paddingHorizontal and paddingVertical props.
